lib/msal-browser/src/cache/DatabaseStorage.ts

```diff
@@ -234,21 +234,23 @@ export class DatabaseStorage<T> implements IAsyncStorage<T> {
         if (this.db && this.dbOpen) {
             this.closeConnection();
         }
-        
-        // @ts-ignore
-        const existingDatabases = await window.indexedDB.databases();
-        const database = existingDatabases.find((database: IDBDatabaseInfo) => database.name === DB_NAME );
-
-        // If database exists, delete it
-        if (database) {
-            return new Promise<boolean>((resolve: Function, reject: Function) => {
-                const deleteDbRequest = window.indexedDB.deleteDatabase(DB_NAME);
-                deleteDbRequest.addEventListener("success", () => resolve(true));
-                deleteDbRequest.addEventListener("error", () => reject(false));
-            });
+
+        try {
+            // @ts-ignore
+            const existingDatabases = await window.indexedDB.databases();
+            const database = existingDatabases.find((database: IDBDatabaseInfo) => database.name === DB_NAME );
```

Comment:
Is this lookup actually needed? It seems like you could blindly invoke `window.indexedDB.deleteDatabase(DB_NAME)` and either the db exists and it gets deleted, or it doesn't exist.

Reply:
I'll keep fiddling with it to see if there's real perf concerns, I originally did the lookup assuming a write was a significantly more expensive operation than a read and find.

Reply:
mdn isn't clear on the perf impact either way.  You'd have to inspect the implementations to be certain.  The notes indicate though that the API won't throw in the DB you're trying to delete doesn't exist.  This implies you shouldn't check for existence first.